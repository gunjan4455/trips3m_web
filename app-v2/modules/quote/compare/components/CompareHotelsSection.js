import React from 'react';
import { PropTypes } from 'prop-types';

import { GHotelRatingDiv } from '../G';
import { StarIcon, } from 'app/helpers/Icon/Icon';

const CompareHotelsSection = ({ hotels }) => {

  return (
    <div className="flexFull p15">
      {
        hotels.map((hotel) => {
          return hotel.hotelInfos.map((hotelInfo) => {
            return (
              <div className="mb24 contentList" key={`hotel_${hotelInfo.hotelId}`}>
                <p className="f14p">{hotelInfo.name}</p>
                <GHotelRatingDiv
                  className="flex alignCenter justifyCenter p2 pl15 pr15 radius2 mt8 text-center">
                  <span className="f14p mr2 at_ratingText">{hotelInfo.rating || '-'}</span>
                  <span className="icon-14">
                    <StarIcon />
                  </span>
                </GHotelRatingDiv>
              </div>
            );
          });
        })
      }
    </div>
  );
};

CompareHotelsSection.propTypes = {
  hotels: PropTypes.array.isRequired
};

export default CompareHotelsSection;
