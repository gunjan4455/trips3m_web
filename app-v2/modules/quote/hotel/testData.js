export const hotelInfo = {
  quoteCityId: 35268058,
  hotelId: 3436,
  tripDayId: 38054617,
  nights: '2',
  city: 'Calangute',
  name: 'Sunkissed Plaza',
  rating: 3,
  address: 'Calangute -  North Goa, Gauravaddo Calangute\r\nBardez GOA 403516 INDIA',
  taUrl: 'http://www.tripadvisor.com/Hotel_Review-g306995-d3916311-Reviews-Sunkissed_Plaza-Calangute_Goa.html',
  dId: 3916311,
  gId: 306995,
  roomType: ' Deluxe ',
  isAlternate: 0,
  comments: '',
  selectionStatus: null,
  description: 'Sunkissed Plaza, Goa, is set in a pleasing location near the blue waters of Calangute Beach. Situated far from the hustle-bustle of the city centre, this hotel offers a stay that is indeed tranquil. Guests can enjoy the beauty of the surroundings from their rooms with the gamut of amenities during the stay. \r\n\r\n<b>Location:</b>\r\nSunkissed Plaza is located at Gauravaddo Calangute Bardez. The Calangute Beach (Approx. 1km), and St Anthony\'s Chapel, Goa (Approx. 1km) are the nearby places of tourist interest from Sunkissed Plaza. Heavily fringed with coconut trees, Calangute Beach offers a spectacular view of the sea. Ideal for numerous water sports and beach fun, Calangute Beach, is a popular tourist destination near Sunkissed Plaza. Ancestral Goa Museum, Carambolim Lake, and Arvalam Waterfalls are the other places that are worth-exploring in Goa.\r\n\r\nDistance from Goa International Airport: 39 km (approx.)\r\nDistance from Thivim Railway Station: 22 km (approx.)\r\n\r\n<b>Hotel Features:</b>\r\nSunkissed Plaza is a ravishing estate surrounded by lush green trees. Simplistic yet comfy, Sunkissed Plaza features well-appointed rooms with all the necessary amenities. Basic amenities offered by Sunkissed Plaza are internet, 24-hour front desk, 24-hour room service, travel desk, 24-hour security and doctor on call. Business services and conference suites are available at Sunkissed Plaza for corporate travellers. There is a swimming pool for a refreshing stay. Wedding and banquet services can be arranged by Sunkissed Plaza if the patrons wish to organise social events. Game lovers can step inside the game room and spend their leisure time. For ultimate rejuvenation and relaxation, patrons can avail of Jacuzzi. Patrons can also sit back and relax at the lounge during the stay. The multi-cuisine restaurant serves tempting meals that gratify the hunger cravings of the guests.\r\n\r\n<b>Rooms:</b>\r\nStandard, premium room and deluxe rooms are available for lodging and boarding. The rooms are cosy with a modern feel to assure an undisturbed stay at Sunkissed Plaza. Elegantly appointed, these room offer a beautiful view of the lush greenery and swimming pool. Air conditioning, colour television, minibar, refrigerator, safe and telephone are the room amenities provided at Sunkissed Plaza.',
  hotelImages: [
    {
      id: 412558,
      url: 'https://res.cloudinary.com/dma3zc8eo/image/fetch/f_auto,q_auto:eco,w_630/https://dknfdeb8v69ms.cloudfront.net/attachments/pictures/412558/original/5296187_21_z.jpg'
    },
    {
      id: 412570,
      url: 'https://res.cloudinary.com/dma3zc8eo/image/fetch/f_auto,q_auto:eco,w_630/https://dknfdeb8v69ms.cloudfront.net/attachments/pictures/412570/original/5296187_22_z.jpg'
    },
    {
      id: 412585,
      url: 'https://res.cloudinary.com/dma3zc8eo/image/fetch/f_auto,q_auto:eco,w_630/https://dknfdeb8v69ms.cloudfront.net/attachments/pictures/412585/original/5296187_31_z.jpg'
    },
    {
      id: 412594,
      url: 'https://res.cloudinary.com/dma3zc8eo/image/fetch/f_auto,q_auto:eco,w_630/https://dknfdeb8v69ms.cloudfront.net/attachments/pictures/412594/original/5296187_35_z.jpg'
    },
    {
      id: 412604,
      url: 'https://res.cloudinary.com/dma3zc8eo/image/fetch/f_auto,q_auto:eco,w_630/https://dknfdeb8v69ms.cloudfront.net/attachments/pictures/412604/original/5296187_36_z.jpg'
    },
    {
      id: 412617,
      url: 'https://res.cloudinary.com/dma3zc8eo/image/fetch/f_auto,q_auto:eco,w_630/https://dknfdeb8v69ms.cloudfront.net/attachments/pictures/412617/original/5296187_26_z.jpg'
    },
    {
      id: 412627,
      url: 'https://res.cloudinary.com/dma3zc8eo/image/fetch/f_auto,q_auto:eco,w_630/https://dknfdeb8v69ms.cloudfront.net/attachments/pictures/412627/original/5296187_32_z.jpg'
    },
    {
      id: 412639,
      url: 'https://res.cloudinary.com/dma3zc8eo/image/fetch/f_auto,q_auto:eco,w_630/https://dknfdeb8v69ms.cloudfront.net/attachments/pictures/412639/original/5296187_34_z.jpg'
    },
    {
      id: 412650,
      url: 'https://res.cloudinary.com/dma3zc8eo/image/fetch/f_auto,q_auto:eco,w_630/https://dknfdeb8v69ms.cloudfront.net/attachments/pictures/412650/original/5296187_20_z.jpg'
    },
    {
      id: 412663,
      url: 'https://res.cloudinary.com/dma3zc8eo/image/fetch/f_auto,q_auto:eco,w_630/https://dknfdeb8v69ms.cloudfront.net/attachments/pictures/412663/original/5296187_28_z.jpg'
    }
  ]
};

export const hotel = {
  days: [
    2
  ],
  hotelInfos: [
    hotelInfo
  ]
};

export const hotels = [
  hotel
];
