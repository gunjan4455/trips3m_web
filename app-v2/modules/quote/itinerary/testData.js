export const itineraryInfo = [
  {
    "0": {
      order: '1',
      title: 'Private Transfer from Airport to Hotel',
      description:
        'A representative will meet and take you to you to your hotel.',
      eventTime: '15:35 - India Standard Time - 2 hours',
      sectionType: 'cab',
      imagesUrl: [
        'https://dailystormer.name/wp-content/uploads/2013/07/London-taxi.jpg',
        'https://dailystormer.name/wp-content/uploads/2013/07/London-taxi.jpg'
      ]
    },
    "1": {
      order: '4',
      title: 'Arrival at Dubai Airport.',
      description:
        'Arrival at Dubai Airport at Evening, meet and greet at Airport and we transfer you toyour hotel. Check in to hotel.',
      eventTime: 'Monday, 24 August, 2019',
      sectionType: 'itinerary',
      imagesUrl: []
    },
    "2": {
      order: '2',
      title: 'hotel details',
      description: 'hotel detail',
      eventTime: 'hotel timing',
      sectionType: 'hotel',
      imagesUrl: [
        'https://dailystormer.name/wp-content/uploads/2013/07/London-taxi.jpg'
      ],
      otherDetails: {
        name: 'hotel',
        rating: 5,
        email: '',
        phone: '44445454',
        web_url: '',
        category: '',
        hotelDesc: 'hotel description',
        tripAdvisorId: '',
        lat: '28.489743',
        long: '77.074602'
      }
    },
    "3": {
      order: '3',
      title: 'Night Safari',
      description:
        "At night, you will be taken for a memorable Night Safari experience. Being the first wildlife night park, it shows you nocturnal animalsâ€™ behavior in their natural habitat. This open-air zoo is known to house nearly 130 species of animals. After an exceptional experience, return to your hotel and stay overnight. Tip: Don't feed animals at the Night Safari",
      eventTime: '15:35 - India Standard Time',
      sectionType: 'attraction',
      imagesUrl: []
    },

    "4": {
      order: '4',
      title: 'Arrival',
      description:
        'Arrival at Dubai Airport at Evening, meet and greet at Airport and we transfer you toyour hotel. Chk in to hotel.',
      eventTime: '15:35 - India Standard Time',
      sectionType: 'flight',
      otherDetails: {
        name: 'prashant',
        image_url:
          'https://dailystormer.name/wp-content/uploads/2013/07/London-taxi.jpg',
        address: 'abc',
        rating: 4.3,
        phone: '33434',
        web_url: '',
        airport_des: 'airport description',
        lat: '28.489743',
        long: '77.074602'
      }
    },
    "5": {
      order: '3',
      title: 'Back to Hotel',
      description: 'going back to hotel after sight seeing',
      eventTime: '15:35 - India Standard Time',
      sectionType: 'attraction',
      imagesUrl: []
    }
  }
];
