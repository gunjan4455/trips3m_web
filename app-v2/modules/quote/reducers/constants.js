import createApiReducer from '../../../reducers/api';
import { LOAD, LOAD_SUCCESS, LOAD_FAIL } from '../actions/constants';

export default createApiReducer([
  LOAD,
  LOAD_SUCCESS,
  LOAD_FAIL
]);

export const getQuoteRejectionReasons = state => (
  (
    state.quote.constants.data && state.quote.constants.data.quoteRejectionReasons
  ) || {}
);

export const getRequestCallbackReasons = state => (
  (
    state.quote.constants.data && state.quote.constants.data.requestCallBackOptions
  ) || {}
);

export const quoteConstantsLoaded = (state) => {
  return state.quote.constants.loaded || false;
};
