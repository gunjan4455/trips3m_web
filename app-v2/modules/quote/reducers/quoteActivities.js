import {
  MORE_QUOTES_LOAD,
  MORE_QUOTES_LOAD_SUCCESS,
  SWITCH_QUOTE,
  UPDATE_REQUESTED_TRIP,
} from '../actions/quoteActivities';

const initialState = {
  currentQuoteId: undefined,
  requestedTripId: undefined,
  moreQuotesRequested: false,
};

export default function reducer(state = initialState, action) {
  switch (action.type) {
    case SWITCH_QUOTE:
      return {
        ...state,
        currentQuoteId: action.quoteId,
      };
    case UPDATE_REQUESTED_TRIP:
      return {
        ...state,
        requestedTripId: action.requestedTripId
      };
    case MORE_QUOTES_LOAD: {
      return {
        ...state,
        moreQuotesRequested: false
      };
    }
    case MORE_QUOTES_LOAD_SUCCESS: {
      return {
        ...state,
        moreQuotesRequested: true
      };
    }
    default:
      return state;
  }
}

export const getTripId = (state) => (
  state.quote.actions.requestedTripId
);

export const getCurrentQuoteId = (state) => (
  state.quote.actions.currentQuoteId
);
