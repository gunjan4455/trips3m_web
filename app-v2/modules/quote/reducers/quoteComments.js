import { LOAD, LOAD_SUCCESS, LOAD_FAIL } from '../actions/quoteComments';

const initialState = {
  loaded: false,
  loading: false,
  data: {}
};

export default function quoteCommentsReducer(state = initialState, action = {}) {

  switch (action.type) {
    case LOAD: {
      return {
        ...state,
        loaded: false,
        loading: true
      };
    }
    case LOAD_SUCCESS: {
      const quoteComments = {};
      action.result.data.results.forEach((comment) => {
        quoteComments[comment.quoteId] = comment;
      });

      return {
        ...state,
        loaded: true,
        loading: false,
        data: quoteComments
      };
    }
    case LOAD_FAIL: {
      return {
        ...state,
        loaded: false,
        loading: false
      };
    }
    default:
      return state;
  }
}
