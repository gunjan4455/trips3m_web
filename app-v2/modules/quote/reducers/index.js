import { combineReducers } from 'redux';

import quoteList from './quoteList';
import quoteActivities from './quoteActivities';
import quoteConstants from './constants';
import pdmReducer from 'modules/pdm/reducer';
import quoteComments from './quoteComments';

export default combineReducers({
  list: quoteList,
  actions: quoteActivities,
  constants: quoteConstants,
  pdms: pdmReducer,
  comments: quoteComments
});
