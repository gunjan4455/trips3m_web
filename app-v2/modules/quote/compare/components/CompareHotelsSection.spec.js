import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

import CompareHotelsSection from './CompareHotelsSection';
import { StarIcon, } from 'app/helpers/Icon/Icon';
import { quote1 } from 'modules/quote/reducers/testData';


configure({ adapter: new Adapter() });

const shallowWrappperCompareHotelsSection = (props) => {
  return shallow(<CompareHotelsSection {...props} />);
};

const DEFAULT_PROPS = {
  hotels: quote1.hotelsDetail,
};

describe('Compare Module', () => {

  let props = { ...DEFAULT_PROPS };

  afterEach(() => {
    props = { ...DEFAULT_PROPS };
  });

  it('should match with the previous snapshot', () => {
    const wrapper = shallowWrappperCompareHotelsSection(props);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render star icon', () => {
    const wrapper = shallowWrappperCompareHotelsSection(props);
    expect(wrapper.find(StarIcon)).toHaveLength(1);
  });

  it('should not render star icon', () => {
    props.hotels[0].hotelInfos[0].rating = 0;
    const wrapper = shallowWrappperCompareHotelsSection(props);
    expect(wrapper.find('.at_ratingText').text()).toBe('-');
  });

});
