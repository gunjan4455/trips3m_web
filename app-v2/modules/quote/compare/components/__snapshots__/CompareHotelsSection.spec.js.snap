// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Compare Module should match with the previous snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CompareHotelsSection
    hotels={
      Array [
        Object {
          "days": Array [
            1,
          ],
          "hotelInfos": Array [
            Object {
              "address": null,
              "city": "Bardez",
              "comments": "",
              "dId": null,
              "description": null,
              "gId": null,
              "hotelId": 359407,
              "hotelImages": Array [],
              "isAlternate": 0,
              "name": "Rio Boutique",
              "nights": "1",
              "quoteCityId": 35268057,
              "rating": 4,
              "roomType": " Deluxe ",
              "selectionStatus": null,
              "taUrl": null,
              "tripDayId": 38054614,
            },
          ],
        },
      ]
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        Array [
          <div
            className="mb24 contentList"
          >
            <p
              className="f14p"
            >
              Rio Boutique
            </p>
            <ForwardRef(render)
              className="flex alignCenter justifyCenter p2 pl15 pr15 radius2 mt8 text-center"
            >
              <span
                className="f14p mr2 at_ratingText"
              >
                4
              </span>
              <span
                className="icon-14"
              >
                <IconSvg />
              </span>
            </ForwardRef(render)>
          </div>,
        ],
      ],
      "className": "flexFull p15",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "hotel_359407",
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <p
              className="f14p"
            >
              Rio Boutique
            </p>,
            <ForwardRef(render)
              className="flex alignCenter justifyCenter p2 pl15 pr15 radius2 mt8 text-center"
            >
              <span
                className="f14p mr2 at_ratingText"
              >
                4
              </span>
              <span
                className="icon-14"
              >
                <IconSvg />
              </span>
            </ForwardRef(render)>,
          ],
          "className": "mb24 contentList",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Rio Boutique",
              "className": "f14p",
            },
            "ref": null,
            "rendered": "Rio Boutique",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                <span
                  className="f14p mr2 at_ratingText"
                >
                  4
                </span>,
                <span
                  className="icon-14"
                >
                  <IconSvg />
                </span>,
              ],
              "className": "flex alignCenter justifyCenter p2 pl15 pr15 radius2 mt8 text-center",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": 4,
                  "className": "f14p mr2 at_ratingText",
                },
                "ref": null,
                "rendered": 4,
                "type": "span",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <IconSvg />,
                  "className": "icon-14",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {},
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                "type": "span",
              },
            ],
            "type": Object {
              "$$typeof": Symbol(react.forward_ref),
              "__emotion_base": "div",
              "__emotion_forwardProp": undefined,
              "__emotion_real": [Circular],
              "__emotion_styles": Array [
                Object {
                  "border": "1px solid #d4d4d4",
                  "width": "48px",
                },
              ],
              "defaultProps": undefined,
              "displayName": "Styled(div)",
              "render": [Function],
              "withComponent": [Function],
            },
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          Array [
            <div
              className="mb24 contentList"
            >
              <p
                className="f14p"
              >
                Rio Boutique
              </p>
              <ForwardRef(render)
                className="flex alignCenter justifyCenter p2 pl15 pr15 radius2 mt8 text-center"
              >
                <span
                  className="f14p mr2 at_ratingText"
                >
                  4
                </span>
                <span
                  className="icon-14"
                >
                  <IconSvg />
                </span>
              </ForwardRef(render)>
            </div>,
          ],
        ],
        "className": "flexFull p15",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "hotel_359407",
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <p
                className="f14p"
              >
                Rio Boutique
              </p>,
              <ForwardRef(render)
                className="flex alignCenter justifyCenter p2 pl15 pr15 radius2 mt8 text-center"
              >
                <span
                  className="f14p mr2 at_ratingText"
                >
                  4
                </span>
                <span
                  className="icon-14"
                >
                  <IconSvg />
                </span>
              </ForwardRef(render)>,
            ],
            "className": "mb24 contentList",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Rio Boutique",
                "className": "f14p",
              },
              "ref": null,
              "rendered": "Rio Boutique",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": Array [
                  <span
                    className="f14p mr2 at_ratingText"
                  >
                    4
                  </span>,
                  <span
                    className="icon-14"
                  >
                    <IconSvg />
                  </span>,
                ],
                "className": "flex alignCenter justifyCenter p2 pl15 pr15 radius2 mt8 text-center",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": 4,
                    "className": "f14p mr2 at_ratingText",
                  },
                  "ref": null,
                  "rendered": 4,
                  "type": "span",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <IconSvg />,
                    "className": "icon-14",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {},
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                  "type": "span",
                },
              ],
              "type": Object {
                "$$typeof": Symbol(react.forward_ref),
                "__emotion_base": "div",
                "__emotion_forwardProp": undefined,
                "__emotion_real": [Circular],
                "__emotion_styles": Array [
                  Object {
                    "border": "1px solid #d4d4d4",
                    "width": "48px",
                  },
                ],
                "defaultProps": undefined,
                "displayName": "Styled(div)",
                "render": [Function],
                "withComponent": [Function],
              },
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;
